src/Storages/System/StorageSystemIcebergHistory.h

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <optional>
+#include <Parsers/ASTIdentifier.h>
+#include <Storages/System/IStorageSystemOneBlock.h>
+
+
+namespace DB
+{
+
+class Context;
+
+/** Implements a table engine for the system "iceberg_history".
```

Comment:
There is no such system, "iceberg_history". Please edit the comment to make it sound correct.